client/app/services/editor.service.js

```diff
@@ -79,6 +80,40 @@
             $window.open(url);
         }
 
+        /**
+         * Launch the iD X editor
```

Comment:
Shouldn't the id X stuff be a separate PR?

Reply:
Missed that one. Fixed with 29d203a